plotextractor/converter.py

```diff
@@ -47,6 +47,21 @@
 from .errors import InvalidTarball
 from .output_utils import get_converted_image_name, get_image_location
 
+MAX_BYTES = 5 * 1024 * 1024  # 5 MB cap
+
+def compress_png(path, image_format):
+    """
+    Runs compress_png on the given file.
+    If it is larger than 5MB, it will be compressed to a PNG
+    with a palette of 256 colors.
+    This approximation based on tests, converts a 15mb png to a 4.5mb png
+    """
+    size = os.path.getsize(path)
+
+    if size > MAX_BYTES:
+        img = Image.open(path)
+        pal = img.convert("P", palette=Image.ADAPTIVE, colors=256)
+        pal.save(path, format=image_format, optimize=True)
```

Comment:
we could add a try catch? also it does it in-place? Meaning it might corrupt the original?